sympy/vector/coordsysrect.py

```diff
@@ -10,7 +10,16 @@
                                     SpaceOrienter, QuaternionOrienter)
 
 
-class CoordSysCartesian(Basic):
+def CoordSysCartesian(*args, **kwargs):
+    SymPyDeprecationWarning(
```

Comment:
It needs to be specified as the `issue` argument. This is a required argument now in the `1.1` branch.